etn-ccis/blui-react-native-workflows
# Feature/5128 registration success screen

Fixes # BLUI-5128



#### Changes proposed in this Pull Request:

- Added registration success screen
- Added test cases
- Added docs
- Updated example 



#### Screenshots / Screen Recording (if applicable)

- 
![Simulator Screenshot - iPhone 12 - 2024-03-05 at 14 34 10](https://github.com/etn-ccis/blui-react-native-workflows/assets/120575281/e3e29c52-659f-4cd6-aaa9-e7e785404e62)




#### To Test:

- cd login-workflow
- yarn start:example
- Open drawer
- Open Registration Provider
- Enter the valid inputs and go through registration workflow
- Finally, you will see the success screen



#### Any specific feedback you are looking for?

-
